Sources/StreamChatUI/CommonViews/Attachments/AttachmentViews/FileAttachmentView.swift

```diff
@@ -5,15 +5,31 @@
 import StreamChat
 import UIKit
 
-public typealias ChatDocumentAttachmentView = _ChatDocumentAttachmentView<NoExtraData>
-
-open class _ChatDocumentAttachmentView<ExtraData: ExtraDataTypes>: _View, ThemeProvider {
-    // MARK: - Properties
+/// A view that displays the file attachment.
+open class FileAttachmentView: _View, AppearanceProvider {
+    open var height: CGFloat = 54
```

Comment:
On the other views, we are not making these properties available. The values are hardcoded in the `setupLayout` and if the customer wants to change it need to override the `setupLayout()`. Because all our constraints are with 900 priority, the customer constraint will override ours.

Reply:
-> slack